server/lib/util.js

```diff
@@ -92,16 +92,19 @@ function requireModuleIfExists(moduleName) {
  */
 function dateFormatter(rows, dateFormat) {
   const DATE_FORMAT = dateFormat || 'YYYY-MM-DD HH:mm:ss';
-
-  _.forEach(rows, element => {
-    _.forEach(element, (value, key) => {
-      if (_.isDate(element[key])) {
-        element[key] = moment(element[key]).format(DATE_FORMAT);
-      }
+  if (_.isArray(rows)) {
+    _.forEach(rows, element => {
+      _.forEach(element, (value, key) => {
+        if (_.isDate(element[key])) {
+          element[key] = moment(element[key]).format(DATE_FORMAT);
+        }
+      });
     });
-  });
 
-  return rows;
+    return rows;
+  }
+
+  return moment(rows).format(DATE_FORMAT);
```

Comment:
Hmm... instead of having one function with different behaviors, it would be better to write two functions and have the programmer choose which one to use - one for arrays and one for single values.

Also, remember unit tests :)